Don't query trades for locations ignored (#5314)
rotki/rotki: docs/changelog.rst

* :bug:`-` Connecting to substrate nodes will no longer timeout prematurely for systems with slow connections.
* :bug:`-` Transfers between tracked accounts will now have a correct label in the UI.
* :bug:`-` Users will be able to finish balance queries if they have assets with missing information.

> * :bug:`-` Visiting the trades page won't query now trades from exchanges that are ignored.

Reviewer: ```suggestion
* :bug:`5265` Visiting the trades page will no longer query trades from ignored exchanges.
```
